newspaper/article.py

```diff
@@ -152,6 +153,23 @@ def build(self):
         self.parse()
         self.nlp()
 
+    def _parse_scheme_file(self, path):
```

Comment:
@dlrobertson what are the use-cases you saw where you'd need to parse `file://` resources instead of the typical `https://`, `http://`, and so on? Did you want to use this lib to crawl offline file-systems of html pages?

I am wondering if this use-case is common

Reply:
> Did you want to use this lib to crawl offline file-systems of html pages? 

Yes

> I am wondering if this use-case is common.

Probably not too common.